$NetBSD: patch-Makefile.in.in,v 1.4 2021/03/28 10:36:33 hauke Exp $

Avoid picking up local config when checking features

Install {c,e}tags man pages with xemacs prefix

Make TAR settable, to work around shortcomings in base tar(1)

--- Makefile.in.in.orig	2015-03-25 11:25:33.000000000 +0000
+++ Makefile.in.in
@@ -65,7 +65,7 @@ LC_ALL = C
 RM = rm -f
 MAKEPATH=./lib-src/make-path
 pwd = /bin/pwd
-TAR = tar
+TAR ?= tar
 
 ## ==================== Things `configure' Might Edit ====================
 
@@ -407,7 +407,7 @@ lwlib/config.h: ${srcdir}/lwlib/config.h
 ## all.  --cet
 
 check-features: all
-	${blddir}/src/${PROGNAME} -batch -l check-features.el
+	${blddir}/src/${PROGNAME} -vanilla -batch -l check-features.el
 
 install-only: ${MAKE_SUBDIR} check-features install-arch-dep install-arch-indep
 
@@ -495,8 +495,10 @@ install-arch-indep: mkdir info
 	fi
 	cd ${srcdir}/etc && \
 	  for page in xemacs etags ctags gnuserv gnuclient gnuattach gnudoit; do \
-	    ${INSTALL_DATA} ${srcdir}/etc/$${page}.1 ${mandir}/$${page}${manext} ; \
-	    chmod 0644 ${mandir}/$${page}${manext} ; \
+	    if test $${page} = etags || test $${page} = ctags ; then \
+	      ${BSD_INSTALL_MAN} ${srcdir}/etc/$${page}.1 ${mandir}/xemacs-$${page}${manext} ; else \
+	      ${BSD_INSTALL_MAN} ${srcdir}/etc/$${page}.1 ${mandir}/$${page}${manext} ; \
+	    fi \
 	  done
 	@echo "If you would like to save approximately 2M of disk space, do"
 	@echo "make gzip-el"
